spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-spi/src/main/java/org/springframework/cloud/data/module/ModuleStatus.java

```diff
@@ -60,6 +60,11 @@
 		deployed,
 
 		/**
+		 * The module exited normally
```

Comment:
Should it read "all modules have ..." ?

Moreover, should this javadoc state that this only applies to tasks (as far as I understand)